pkg/core/interop/context.go

```diff
@@ -155,6 +160,14 @@ func (ic *Context) GetFunction(id uint32) *Function {
 	return nil
 }
 
+// BaseExecFee represents factor to multiply syscall prices with.
+func (ic *Context) BaseExecFee() int64 {
+	if ic.Chain == nil {
```

Comment:
Is it possible (not in tests)?

Reply:
I don't think it is, but it could be so as not every interop requires chain.

Reply:
Have you tried running without this check? C# node is a bit more problematic wrt this because it specifically doesn't have a proper snapshot in stateless verification context. But we do and it shouldn't be a problem for us, any real context in neo-go has it.

Reply:
I have tried, multiple tests are failing (in `core/interop`, `compiler` and `smartcontract` packages). Fixing this without duplication requires dealing with import cycles.